bumble/device.py

```diff
@@ -1679,14 +1733,18 @@ async def start_extended_advertising(
         Returns:
           Handle of the new advertising set.
         """
+        if self.legacy_advertiser:
+            logger.warning(
+                'Trying to start Legacy and Extended Advertising at the same time!'
+            )
 
         adv_handle = -1
         # Find a free handle
         for i in range(
             DEVICE_MIN_EXTENDED_ADVERTISING_SET_HANDLE,
             DEVICE_MAX_EXTENDED_ADVERTISING_SET_HANDLE + 1,
         ):
-            if i not in self.extended_advertising_handles:
+            if i not in self.extended_advertisers.keys():
```

Comment:
nit: `if i not in self.extended_advertisers` should be sufficient.

Reply:
I think this could provide a better readability?

Reply:
Either way is fine, but in general I think it is more readable to use the standard/idiomatic way of using collections.
For looking up a key in a dict, the recommended way in most style guides is to just use `key in dict` (see https://google.github.io/styleguide/pyguide.html#284-decision for example).